modules/apps/27-interchain-accounts/controller/keeper/msg_server_test.go

```diff
@@ -0,0 +1,88 @@
+package keeper_test
+
+import (
+	sdktypes "github.com/cosmos/cosmos-sdk/types"
+
+	icatypes "github.com/cosmos/ibc-go/v5/modules/apps/27-interchain-accounts/controller/types"
+	channeltypes "github.com/cosmos/ibc-go/v5/modules/core/04-channel/types"
+	host "github.com/cosmos/ibc-go/v5/modules/core/24-host"
+	ibctesting "github.com/cosmos/ibc-go/v5/testing"
+)
+
+func (suite *KeeperTestSuite) TestRegisterAccount() {
```

Comment:
@colin-axner some of tests are the same or very similar as the existing tests for `RegisterInterchainAccount` such as [this one](https://github.com/cosmos/ibc-go/blob/af4e6516b7826e7b8fcb062a3ea4608b4bdaf532/modules/apps/27-interchain-accounts/controller/keeper/account_test.go#L25-L33) let me know if you think it's overkill to have the test in multiple locations.

Reply:
When we deprecate `RegisterInterchainAccount` in favor of the message server API, we can just remove the others. I think it is fine to have duplicates for now